api-server/src/server/index.js

```diff
@@ -89,6 +82,31 @@ app.start = _.once(function () {
   });
 });
 
+if (sentry.dsn === 'dsn_from_sentry_dashboard') {
+  log('Sentry reporting disabled unless DSN is provided.');
+} else {
+  Sentry.init({
+    dsn: sentry.dsn,
+    integrations: [
+      new Sentry.Integrations.Http({ tracing: true }),
+      new Tracing.Integrations.Express({
+        app
```

Comment:
This wasn't request for a change, sorry, just an observation.

Reply:
Yup - that was the first thing I tested, and it did the job, but I dug deeper just in case and figured it's better we delay and let the "app" be "baked" with goodies in the bootstrap phase before we init to avoid any surprises elsewhere.

Reply:
Cool, cool. Sounds good!